.github/workflows/release.yml

```diff
@@ -45,6 +45,7 @@ jobs:
         run: echo ::set-output name=tag::${GITHUB_REF#refs/*/}
 
       - run: npm ci
+      - run: npm run build
```

Comment:
- run: npm run prepublishOnly

Reply:
Adjusted! Do we need an RC3 to try this out, or do you have enough to test as-is?